### PR TITLE
Document the new GraphQL-based CLI

### PR DESCRIPTION
**Pull request by chrzaszcz**
- Introduce the CLI in "Quick Setup"
- Replace examples of old commands with the new ones.

The GraphQL API itself needs more docs as well - there are examples with REST API that can be converted to GraphQL.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/133551) / small_tests / 6ac25dc565b305622fb57a912ec8ccf7af09256b
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3739/133551/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3739/133551/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-24_13.41.41/lib.mongooseim.logs/run.2022-08-24_13.41.41/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/133576) / small_tests / a552c2606cc22162a6cc53933ac26cb36da26c33
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3739/133576/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3739/133576/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-24_13.56.22/lib.mongooseim.logs/run.2022-08-24_13.56.23/suite.log.html)

File: doc/getting-started/Quick-setup.md [hunk 1]
```diff
@@ -76,76 +76,157 @@ The default XMPP domain served by MongooseIM right after installation is `localh
...
+        "bob@localhost",
+        "carol@localhost",
+        "dan@localhost"
+      ]
+    }
+  }
+}
+```
+
+If you want to delete users in your host:
```
Comment (JanuszJakubiec) [inline doc/getting-started/Quick-setup.md:126]:
```suggestion
If you want to delete user in your host:
```
```suggestion
If you want to delete users in your host:
```

Comment (JanuszJakubiec) [inline doc/getting-started/Quick-setup.md:126]:
```suggestion
If you want to delete user in your host:
```

Comment (JanuszJakubiec) [reply doc/getting-started/Quick-setup.md:126]: In my opinion using "users" suggest possibility of deleting multiple users using that query.

**Review by JanuszJakubiec — APPROVED** [review]
Looks good, I've added one suggestion.

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/133727) / small_tests / b094e91eff39b0780442814b82c7fcb6646bb145
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3739/133727/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3739/133727/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-08-25_06.44.38/lib.mongooseim.logs/run.2022-08-25_06.44.38/suite.log.html)

**Pull request merged**